src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const TabButtonStyle = () => css`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabButton = styled(Button)`
+    ${TabButtonStyle}
+`;
+
+const TabsStyle = () => css`
+    @media (min-width: 1048px) {
+        button:first-child {
+            margin-right: 73%;
```

Comment:
Curious where does this 73% come from? Was only doing this above 1023px Mike's idea?

Reply:
73% is the maximum margin and 1048 is the screen size ,I could go without squishing the tab contents.Definitely want your suggestions on this.

Reply:
What about something like this in `TabsStyle` css?:

```
justify-items: flex-end;
@media (min-width: 1048px) {
    button:first-of-type {
        margin-right: auto;
    }
}
```

Reply:
I don't need the min width any more. It work's better without the min-width.